### PR TITLE
As per request https://github.com/broadinstitute/picard/issues/519 th…

### PR DESCRIPTION
**Pull request by kasont**
### Description

Please explain the changes you made here.
Explain the **motivation** for making this change. What existing problem does the pull request solve?

**As per https://github.com/broadinstitute/picard/issues/519 wants, the output is changed from exception to null.**

### Checklist

- [ ] Code compiles correctly
- [ ] New tests covering changes and new functionality
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary
- [ ] Is not backward compatible (breaks binary or source compatibility)

…e output is null.

### CODE REVIEW

File: src/main/java/htsjdk/variant/variantcontext/VariantContext.java [hunk 1]
```diff
@@ -1695,14 +1684,20 @@ public static boolean hasSymbolicAlleles( final List<Allele> alleles ) {
...
 
-        return getAlternateAlleles().stream()
-                .map(allele -> new Tuple<>(allele, getCalledChrCount(allele)))
-                .max((alleleAndCount1, alleleAndCount2) -> Integer.compare(alleleAndCount1.b, alleleAndCount2.b))
-                .get()
-                .a;
+            return getAlternateAlleles().stream()
+                    .map(allele -> new Tuple<>(allele, getCalledChrCount(allele)))
+                    .max((alleleAndCount1, alleleAndCount2) -> Integer.compare(alleleAndCount1.b, alleleAndCount2.b))
+                    .get()
```
Comment (yfarjoun) [inline src/main/java/htsjdk/variant/variantcontext/VariantContext.java]: instead of the try/catch, I'd use a getOrElse(null)

**Comment by droazen** [comment]
@kasont Was this branch abandoned? Do you still want a review?

File: src/main/java/htsjdk/variant/variantcontext/VariantContext.java [hunk 2]
```diff
@@ -1695,14 +1684,15 @@ public static boolean hasSymbolicAlleles( final List<Allele> alleles ) {
...
-        return getAlternateAlleles().stream()
-                .map(allele -> new Tuple<>(allele, getCalledChrCount(allele)))
-                .max((alleleAndCount1, alleleAndCount2) -> Integer.compare(alleleAndCount1.b, alleleAndCount2.b))
-                .get()
-                .a;
+            return getAlternateAlleles().stream()
+                    .map(allele -> new Tuple<>(allele, getCalledChrCount(allele)))
+                    .max((alleleAndCount1, alleleAndCount2) -> Integer.compare(alleleAndCount1.b, alleleAndCount2.b))
+                    .orElse(null) 
+                    .a;
```
Comment (droazen) [inline src/main/java/htsjdk/variant/variantcontext/VariantContext.java]: Potential `NullPointerException` here

Comment (lbergelson) [inline src/main/java/htsjdk/variant/variantcontext/VariantContext.java]: I think what you want is something like:
```
  return getAlternateAlleles().stream()
                .map(allele -> new Tuple<>(allele, getCalledChrCount(allele)))
                .max((alleleAndCount1, alleleAndCount2) -> Integer.compare(alleleAndCount1.b, alleleAndCount2.b))
                .map( tuple -> tuple.a)
                .orElseGet(null);
```

**Pull request closed without merging**